src/notifications/container.js

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { bindActionCreators } from 'redux'
+
+import * as actions from './actions'
+import * as selectors from './selectors'
+import NotificationList from './components/NotificationList'
+import Notification from './components/Notification'
+import StatusHeading from './components/StatusHeading'
+
+class NotificationContainer extends Component {
+    static propTypes = {
+        unreadNotificationList: PropTypes.arrayOf(PropTypes.object).isRequired,
+        readNotificationList: PropTypes.arrayOf(PropTypes.object).isRequired,
+        init: PropTypes.func.isRequired,
+        showMoreIndex: PropTypes.string,
+        handleToggleShowMore: PropTypes.func.isRequired,
+        handleTick: PropTypes.func.isRequired,
+    }
+
+    componentDidMount() {
+        this.props.init()
+    }
+
+    isNotificationTruncated(message) {
+        const NotificationCharLimit = 150
```

Comment:
This shouldn't start with uppercase. Also, this value is a constant value that you're only using as a length limit for messages (you're also reusing and redeclaring it in `truncateText` method). In React containers I would recommend setting this as a prop with a corresponding entry in `static defaultProps` setting it to the `150` value. Then it acts as your constant, but if we want to re-use it later but change the limit, it can be done without altering this code.

Also the below condition can be simplified to `return message.length > this.props.messageCharLimit`